ui: dropdown outer size

vladlos opened the pull request:
changed max height of dashboard dropdown to fit all items

Resolves: #44892

Release note (ui): none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/45084)
<!-- Reviewable:end -->

dhartunian commented on pkg/ui/src/views/shared/components/dropdown/dropdown.styl:
```diff
@@ -121,5 +121,8 @@ $dropdown-hover-color = darken($background-color, 2.5%)
         background-color $dropdown-hover-color
+.dropdown.full-size
+  .Select-menu-outer, .Select-menu
+    max-height 450px !important
```
can we find a way to do this without using !important

dhartunian commented on pkg/ui/src/views/shared/components/dropdown/index.tsx:
```diff
@@ -107,7 +109,7 @@ export default class Dropdown extends React.Component<DropdownOwnProps, {}> {
       </span>
       {content ? content : <Select
-        className="dropdown__select"
+        className="dropdown__select is-focused is-open"
```
Why are these additional classes necessary here?

vladlos committed:
ui: dropdown outer size

changed max height on dashboard dropdown

Resolves: #44892

Release note (ui): none

dhartunian commented:
bors r+
